test_xdb/models/under_test/clone_schema_tables_data_equality_test.sql

```diff
@@ -0,0 +1,89 @@
+--depends_on: {{ ref('clone_schema_test') }}
```

Comment:
This model shows us counts of not matched rows between original and copied tables/views in test schemas.